src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,116 @@
+  const isMobile = screenWidth === "mobile";
+
+  return (
+    <footer
+      className={`flex w-full flex-col bg-wheat p-[80px] gap-[40px] lg:p-[60px] md:p-[40px] sm:p-[16px] sm:gap-[26px] ${
+        isMobile ? "p-[16px] gap-[26px]" : "p-[80px] gap-[40px]"
+      }`}
+    >
+      <div
+        className={`flex flex-row gap-[287px] lg:gap-[250px] md:gap-[140px] sm:gap-[26px] sm:flex-col ${
+          isMobile ? "flex-col gap-[26px]" : "flex-row gap-[287px]"
+        }`}
+      >
+        <div
+          className={`flex flex-col gap-[40px] sm:gap-[26px] ${
+            isMobile ? "gap-[26px]" : "gap-[40px]"
+          }`}
+        >
+          <div className="flex flex-col gap-[24px]">
+            <p
+              className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center ${
+                isMobile ? "text-center" : "text-left"
+              }`}
+            >
+              Expert Event Solutions
+            </p>
+            <a
+              className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center cursor-pointer ${
+                isMobile ? "text-center" : "text-left"
+              }`}
+              onClick={() => console.log("clicked address")}
+            >
+              260 Daytona Blvd, Daytona <br /> Beach, Florida 32114, United{" "}
+              <br /> States
+            </a>
+            <p
+              className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center cursor-pointer ${
+                isMobile ? "text-center" : "text-left"
+              }`}
+              onClick={() => console.log("clicked email")}
+            >
+              Info@experteventsolutions.com
+            </p>
+          </div>
+        </div>
+
+        <div className="flex flex-col gap-[24px]">
+          <p
+            className={`font-text text-paragraph-sm sm:hidden ${
+              isMobile ? "hidden" : "block"
+            }`}
+            onClick={() => console.log("clicked contacts")}
+          >
+            Contacts
+          </p>
+          <p
+            className={`font-text text-paragraph-sm sm:hidden ${
+              isMobile ? "hidden" : "block"
+            }`}
+            onClick={() => console.log("clicked rental conditions")}
+          >
+            Rental Conditions
+          </p>
+
+          <div
+            className={`flex flex-wrap flex-col gap-[10px] sm:content-center ${
+              isMobile ? "content-center" : "content-start"
+            }`}
+          >
+            <p
+              className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center ${
+                isMobile ? "text-center" : "text-left"
+              }`}
+            >
+              Social Media
+            </p>
+            <div className="flex flex-row gap-[10px] cursor-pointer">
```

Comment:
hey @XDRO I did notice that when hovering over the gaps between the social media images, the cursor: pointer is visible. I would recommend taking out the the gap and give each image a div tag with a margin right of 10px and a cursor pointer as well. Here is an example of what i did: 
<img width="1440" alt="Screenshot 2024-09-13 at 4 47 35 PM" src="https://github.com/user-attachments/assets/e1539f6f-0bed-476b-8faf-8638ab4d768d">